portfolio/src/main/java/com/google/DataServlet.java

```diff
@@ -24,30 +24,34 @@
  
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data") 
-public class DataServlet extends HttpServlet {
-
-  private List<String> phrases;
+public class DataServlet extends HttpServlet 
+{ 
+  private List<String> comments;
 
    @Override
   public void init() {
-    phrases = new ArrayList<String>();
-    phrases.add("Writer");
-    phrases.add("Achiever");
-    phrases.add("Engineer");
-    phrases.add("Student");
-    phrases.add("Visionary");
+    comments = new ArrayList<String>();
   }
 
    @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJsonUsingGson(phrases);
+    String json = convertToJsonUsingGson(comments);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+
+   @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String message = request.getParameter("user-comment");
+    if(message == null || message.length() != 0){
+        comments.add(message);
+    }
+    response.sendRedirect("/index.html#comments-panel");
+  }
  
-  private String convertToJsonUsingGson(List<String> comments) {
+  private String convertToJsonUsingGson(List<String> content) {
```

Comment:
Think the fact that you are using `Gson` is an implementation detail and that `convertToJson` would be sufficient for the title.

For example, what if you need to refactor this later to use a different JSON lib?

Reply:
I hadn't thought of that, I'll change the function name